doc/cephfs/quota.rst

```diff
@@ -74,6 +74,13 @@ To set a quota::
   setfattr -n ceph.quota.max_bytes -v 100000000 /some/dir     # 100 MB
   setfattr -n ceph.quota.max_files -v 10000 /some/dir         # 10,000 files
 
+``ceph.quota.max_bytes`` can also be set using human-friendly units
+(both SI prefixes and the IEC prefixes i.e. K, M, ... E and Ki, Mi, ... Ei
+respectively)::
```

Comment:
Given the conversation [here](https://github.com/ceph/ceph/pull/49008#issuecomment-1410042786), this line needs to be removed.

cc @lxbsz @vshankar @batrick

Reply:
sure